vng_api_common/serializers.py

```diff
@@ -264,3 +264,34 @@ def update(self, instance, validated_data):
                 continue
             setattr(instance, name, validated_data.pop(name))
         return super().update(instance, validated_data)
+
+
+class LengthHyperlinkedRelatedField(serializers.HyperlinkedRelatedField):
+    def __init__(self, *args, **kwargs):
+        self.min_length = kwargs.pop("min_length", None)
+        self.max_length = kwargs.pop("max_length", None)
+        kwargs.setdefault("allow_null", True)
+
+        super().__init__(*args, **kwargs)
+
+    def to_internal_value(self, data):
+        if self.max_length and len(data) > self.max_length:
+            self.fail("max_length", max_length=self.max_length, length=len(data))
```

Comment:
where are these error messages defined?

Reply:
whoops, went a bit too hard on the copypaste